src/mgmt/srv_target.c

```diff
@@ -1075,6 +1087,14 @@ ds_mgmt_hdlr_tgt_create(crt_rpc_t *tc_req)
 	if (tca.tca_newborn != NULL) {
 		struct vos_pool_arg vpa = {0};
 
+		/* clear any orphaned blobs that are associated with this uuid */
+		rc = clear_vos_pool(tc_in->tc_pool_uuid);
+		if (rc != 0) {
+			D_ERROR(DF_UUID ": failed to clear orphan blobs: " DF_RC "\n",
```

Comment:
here you may want to change the D_ERROR() to a new macro DL_ERROR(rc, "failed to clear orphan blobs") (no need for DF_RC, \n, and DP_RC(). We are incrementally phasing out D_ERROR() calls.